eng/Packages.Data.props

```diff
@@ -84,9 +84,9 @@
     <PackageReference Update="Azure.Messaging.ServiceBus" Version="7.2.1" />
     <PackageReference Update="Azure.Messaging.WebPubSub" Version="1.0.0-beta.2" />
     <PackageReference Update="Azure.Identity" Version="1.4.0" />
-    <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.0.2" />
-    <PackageReference Update="Azure.Security.KeyVault.Keys" Version="4.0.2" />
-    <PackageReference Update="Azure.Security.KeyVault.Certificates" Version="4.0.2" />
+    <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.2.0" />
```

Comment:
Why is this being changed?

Reply:
Want to pick up a bug fix for the extension package.